File: week1-iteration-exercises/02_Tues/is_factor_of.py
def getInput():
    number = input("Please enter a positive integer: ")
    return number

def get_factors():
    number = int(getInput())
    cur_num = 1
    while cur_num < number:
        if number % cur_num == 0:
            print(f"{cur_num}: factor of {number}")
        else:
            print(f"{cur_num}: not a factor of {number}")
        cur_num += 1

get_factors()

File: week1-iteration-exercises/02_Tues/IsFactorOf.java
import java.util.Scanner;

class IsFactorOf {
    static Scanner reader = new Scanner(System.in);

    public static Integer getNum() {
        System.out.println("Enter a positive integer: ");
        int num = reader.nextInt();
        reader.close();
        return num;
    }

    public static void main(String[] args) {
        FindFactors.getFactors2(getNum());
        // FindFactors.getFactors(getNum());
    }
}